frontend/src/sdk/useAppConfiguration.ts

```diff
@@ -8,18 +8,9 @@ import { useChrome } from '@redhat-cloud-services/frontend-components/useChrome'
 
 type AppConfigurations = React.ComponentProps<typeof AppInitSDK>['configurations'];
 
-// TODO: remove once RHCLOUD-24173 is fixed and return just `wss://${location.host}${prefix || '/wss/k8s'}`
 const getWsURL = ({ wsPrefix, pathPrefix }: { wsPrefix?: string; pathPrefix?: string } = {}) => {
   const prefix = (wsPrefix || pathPrefix || '') as string;
-  if (localStorage.getItem('hac/proxy-ws') !== null) {
-    return `wss://${location.host}${prefix || '/wss/k8s'}`;
-  }
-
-  if (location.origin.includes('console.dev.redhat.com')) {
-    return `wss://api-toolchain-host-operator.apps.stone-stg-host1.hjvn.p1.openshiftapps.com${prefix}`;
-  }
-
-  return `wss://api-toolchain-host-operator.apps.appstudio-stage.x99m.p1.openshiftapps.com:443${prefix}`;
+  return `wss://${location.host}${prefix || '/wss/k8s'}`;
```

Comment:
I think we'd want to configure the prefix `/wss/k8s` just for stonesoup and leave the default as no prefix? But we can look at that as a follow up.

Reply:
Good point. I think we assume that we are running on consoledot and this path will always be the fallback. We can update this in a follow up to make `/wss/k8s` specific to just stonesoup.